src/components/MoreInformation.js

```diff
@@ -53,12 +55,100 @@ function ExtraData({ data }) {
     } else if (typeof data == "object") {
         let elements = [];
         for (const key in data) {
-            elements.push(
-                <div className={classes.extraData}>
-                    <b>{key}:</b> {data[key]}
-                </div>
-            );
+            //formats additional information instead of just adding it to the more information section
+            if (key === "Additional Information") {
+                const currentData = data[key];
+                if (currentData) {
+                    var finalData = parseMD(currentData);
+                    //adds the parsed data as straight html
+                    elements.push(
+                        <div
+                            className={classes.extraData}
+                            dangerouslySetInnerHTML={{
+                                __html: `<b>${key}:</b> ${finalData}`,
+                            }}
+                        />
+                    );
+                }
+            } else {
+                elements.push(
+                    <div className={classes.extraData}>
+                        <b>{key}:</b> {data[key]}
+                    </div>
+                );
+            }
         }
         return elements;
     }
 }
+
+function parseMD(currentData) {
+    //encodes angle brackets to escape data
+    let workingData1;
+    let workingData2;
+    let indexOfLAB = currentData.indexOf("<");
+    let indexOfRAB = currentData.indexOf(">");
+    if (indexOfLAB !== -1) {
+        workingData1 = currentData.replaceAll("<", "&#60;");
+    } else {
+        workingData1 = currentData;
+    }
+    if (indexOfRAB !== -1) {
+        workingData2 = workingData1.replaceAll(">", "&#62;");
+    } else {
+        workingData2 = workingData1;
+    }
```

Comment:
This is a good start but think we're still vulnerable to a Cross-Site Scripting attack here. Here's the [OWASP Cheat Sheet](https://cheatsheetseries.owasp.org/cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.html#rule-1-html-encode-before-inserting-untrusted-data-into-html-element-content) for preventing XSS attacks. Rule 1 pertains to what we're doing, inserting into plain HTML elements. There are a few characters we need to look out for, and the cheat sheet recommends using a reliable library if possible.

If we can use a library, [DOMPurify](https://github.com/cure53/DOMPurify) seems like a well maintained, lightweight option. If we want to roll our own, let's make sure we cover at least the characters suggested on the cheat sheet: `&, <, >, ", '`.

@livgust do you have a preference here?